examples/show_data.py

```diff
@@ -1,52 +1,62 @@
+import click
 import napari
 
 import pandas as pd
 import numpy as np
 
-from grace.base import GraphAttrs
-from grace.io.image_dataset import mrc_reader
 from pathlib import Path
 
+from grace.base import GraphAttrs
+from grace.io.image_dataset import FILETYPES
 
-# Expects the image data & H5 node positions in the same folder.
-# Use identical naming convention for files & specify whole path to mrc file:
-# e.g. /Users/kulicna/Desktop/dataset/shape_squares/MRC_Synthetic_File_000.mrc
 
-IMAGE_PATH = Path(
-    input(
-        "Enter absolute path to your file "
-        "(e.g. /Users/path/to/your/data/image.mrc, omit ''): "
-    )
-)
-NODES_PATH = Path(str(IMAGE_PATH).replace(".mrc", ".h5"))
+# Define a click command to input the file name directly:
+@click.command(name="Napari Annotator")
+@click.option("--image_path", type=click.Path(exists=True))
```

Comment:
It would be helpful to document the option with a "help" description as shown here: https://click.palletsprojects.com/en/8.1.x/documentation/

Reply:
Agreed - helps with clarity, added in the upcoming commit 🚀